### ISSUE TITLE
Display bug when large amounts appear in the recurrent transactions page footer

### ISSUE THREAD

**Issue by shinebrillant** [enrique-lozano/Monekin#245]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/enrique-lozano/Monekin/blob/main/CONTRIBUTING.md#reporting-issues).
- [X] I agree to follow the [Code of Conduct](https://github.com/enrique-lozano/Monekin/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/enrique-lozano/Monekin/issues?q=is%3Aissue) that already report this problem, without success.
- [X] I have updated the app to the last released version.

### Current Behavior

Originally I had four recurring transactions.

![ResizedImage_2024-11-11_20-23-04_1270](https://github.com/user-attachments/assets/28e90663-2750-4d40-9bec-dc5f52122e37)

If there are many digits in the number, it will be compressed into the text display space.
This amount that is too long will block other items from being displayed.

![Screenshot_20241111-200351](https://github.com/user-attachments/assets/51106896-fa95-4a59-98ca-0206bf072eb4)

### Expected Behavior

Just move the amount below the text to display it.
The description and amount can be displayed on separate lines and can be displayed in full without abbreviation.

```
Total expense per period
* without ...
every day or every year.
total cost
```

![ResizedImage_2024-11-12_20-56-07_1292](https://github.com/user-attachments/assets/72cfe57c-0d6b-4a7f-81b6-2bfebc02f2ad)

### Steps to Reproduce

Add a multi-digit recurrent transaction.

**Comment by enrique-lozano** [comment]
Could you upload a screenshot with the problem? I'm not really sure of what do you mean

**Comment by vincius0000** [comment]
It makes sense to fix this for currencies where inflation is very high.

For such cases instead of showing full number, I might show 3.2M instead of 3,200,000.

To see the actual number you could simply click on transaction.

For any other case, big numbers shouldn't cause such big changes in size, therefore we may have to add clipping

@enrique-lozano what do you think?

**Comment by enrique-lozano** [comment]
Yep, I think that for this problem we need to format the amount number only. As @vincius0000 said, we need to convert 3,200,000 to 3,2M and 6,767.92 to 6,76K.

I think that this simple fix should resolve the issue

**Comment by enrique-lozano** [comment]
Closed with #267 🚀
